packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/configuration/ViewPaletteToolsConfiguration.java

```diff
@@ -18,15 +18,15 @@
 import org.eclipse.sirius.components.core.api.IURLParser;
 import org.eclipse.sirius.components.view.emf.IViewRepresentationDescriptionPredicate;
 import org.eclipse.sirius.components.view.emf.IViewRepresentationDescriptionSearchService;
-import org.springframework.stereotype.Service;
+import org.springframework.context.annotation.Configuration;
 
 /**
  * Bundles the common dependencies that view tool services need into a single object for convenience.
  *
  * @author frouene
  */
-@Service
-public class ViewReconnectionToolsExecutorParameters {
+@Configuration
```

Comment:
Not sure why this change.
`@Configuration`'s Javadoc says:

> Indicates that a class declares one or more `@Bean` methods and may be processed by the Spring container to generate bean definitions and service requests for those beans at runtime

But this does not define any `@Bean`, it just bundles existing ones into a single _parameter object_.

Reply:
Done